lib/exception_hunter/error_creator.rb

```diff
@@ -1,14 +1,16 @@
 module ExceptionHunter
   class ErrorCreator
+    TAGS = %w[HTTP Worker Manual].freeze
+
     class << self
-      def call(**error_attrs)
+      def call(tag = 'HTTP', **error_attrs)
```

Comment:
Maybe a named argument would be better
```suggestion
      def call(tag: 'HTTP', **error_attrs)
```